Makefile

```diff
@@ -1,3 +1,8 @@
+PWD=$(shell pwd)
+export C_INCLUDE_PATH=${PWD}/extern/tmp
+export LIBRARY_PATH=${PWD}/extern/tmp
+export LD_LIBRARY_PATH=${PWD}/extern/tmp
+
```

Comment:
BTW, if user don't install rocksdb in this path but in system path, or /extern/tmp is not exist, we may meet an error like `ld: warning: directory not found for option`